bin/release_automation.sh

```diff
@@ -48,6 +48,9 @@ npx json -I -f package.json -e "this.version='$VERSION_NUMBER'" || { echo "Error
 # Set version number in react-native-editor package.json
 npx json -I -f gutenberg/packages/react-native-editor/package.json -e "this.version='$VERSION_NUMBER'" || { echo "Error: could not update version in react-native-editor package.json"; exit 1; }
 
+# Make sure podfile is updated
+npm run core preios
```

Comment:
It doesn't feel right to me to run this after we bump the version numbers, but before we commit those bumped to the version numbers. Perhaps we should move this either (a) before the version numbers are bumped or (b) after the version numbers are committed. Then we could also add a step that checks if there were any podfile changes and, if there were, commits them separately?

Reply:
@mchowning Do you feel we should always commit separately the changes independently of doing a) or b)?

Reply:
I think it would be nice to have a separate commit for each distinct stage in the release process in order to make it easier to look back and see exactly what the release script did at each step (i.e., when something goes wrong). I don't think this is a critical change that needs to block this PR though, so I'll go ahead and approve. There are a couple of release script updates I'm planning to work on today, so I'll maybe see about separating out the commits in that PR and then you can give it a look and see what you think.